blockstack_client/client.py

```diff
@@ -149,36 +124,40 @@ def register_storage(storage_impl, conf):
     return rc
 
 
-def get_analytics_key( uuid, proxy=None ):
+def get_analytics_key(uuid, proxy=None):
     """
     Get the analytics key from the blockstack server
     """
-    if proxy is None:
-        proxy = get_default_proxy()
 
-    key = None
-    if os.environ.get("BLOCKSTACK_TEST_ANALYTICS_KEY", None) is not None:
-        key = {'analytics_key': os.environ.get("BLOCKSTACK_TEST_ANALYTICS_KEY")}
+    key = os.environ.get('BLOCKSTACK_TEST_ANALYTICS_KEY')
 
-    else:
-        try:
-            key = proxy.get_analytics_key( uuid )
-        except Exception, e:
-            log.debug("Failed to get analytics key")
-            return None
+    if key:
+        return key
 
+    try:
+        proxy = proxy or get_default_proxy()
+        # TODO: proxy does not have a get_analytics_key function.
```

Comment:
`proxy` here should be an instance of `BlockstackRPCClient`, which in turn dispatches method calls to a class that inherits from `xmlrpclib.ServerProxy`.  Method binding is late here--`get_analytics_key` is an RPC method on the server.